schema/deploy/shipping/views.sql

```diff
@@ -844,4 +844,95 @@ comment on view shipping.hcov19_observation_v1 is
   'Custom view of hCoV-19 samples with presence-absence results and best available encounter data';
 
 
+create or replace view shipping.scan_return_results_v1 as
+
+    with status_codes as (
+        -- Create status codes as required by Lab Med's results portal
+        with x(status_code, status) as (
+            values
+            ('DET', 'positive result, colored red + bold'),
+            ('INCON', 'inconclusive, colored orange; treated as positive with respect to information given'),
+            ('NDET', 'not detected/negative, colored blue'),
+            ('pending-other', null),
```

Comment:
I'm currently treating `pending-other` as the scenario where a sample tests positive but the participant has not yet been contacted. No sample in this view (as it stands) will ever contain the status `pending-other`.